### PR TITLE
camelCase noty_notification.js

### LINKED ISSUE #9149: camelCase noty_notification.js filename
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### ❓ Why are we doing this?

See Issue #9140 for more information!

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** these files below:

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below are some "diffs" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add for each file:

`/app/assets/javascripts/application.js`
```diff
+ //= require notyNotification.js
- //= require noty_notification.js
```

`/app/assets/javascripts`:
```diff
+ notyNotification.js
- noty_notification.js.js
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by akshay4n**
Fixes #9149  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by akshay4n** [comment]
@noi5e I changed the name of the file

**Comment by noi5e** [comment]
@akshay4n Great, it looks like you made both changes, let's see how the tests do.

Can you please change the `#0000` to `#9149` here in your comment above? `Fixes #0000 (<=== Add issue number here)`

**Comment by akshay4n** [comment]
@noi5e Do I need to rectify the errors?

**Comment by noi5e** [comment]
> Great, it looks like you made both changes, let's see how the tests do.

@akshay4n Oops, looks like I spoke too soon. I took a closer look at the changes you made.

This line:
https://github.com/publiclab/plots2/blob/e2600941a6f683cddad21f70b9ada56423c92951/app/assets/javascripts/notyNotification.js#L12

Actually belongs in `application.js`, not `notyNotification.js`. Can you make these 2 changes below please?

`/app/assets/javascripts/notyNotification.js`:
```diff
- //= require noty_notification.js
```

`/app/assets/javascripts/application.js`:
```diff
+ //= require notyNotification.js
- //= require noty_notification.js
```

File: app/assets/javascripts/notyNotification.js
Comment (noi5e) [inline app/assets/javascripts/notyNotification.js:12]:
```suggestion
```

**Comment by noi5e** [comment]
Hi @akshay4n, this pull request is very close! Let's just finish it here. Please don't open another pull request!

The only thing we need to do is remove the comment in `notyNotification.js`. Here's a step-by-step on how to do that:

1. Scroll up to the top and click on "Files changed". Screen Shot 2021-02-22 at 11 19 02 AM
2. Scroll down to `notyNotification.js`
3. Click on the three dots
4. Click on "Edit file" 
Screen Shot 2021-02-22 at 11 19 22 AM
5. Delete this line completely: `//= require noty_notification.js`
6. Scroll down, and under "Commit changes," enter a commit message for your commit (something simple like "Deleted comment")
Screen Shot 2021-02-22 at 11 22 33 AM
7. Click on the green Commit changes button.
8. Wait for the tests to run.

You may see some of the tests failing. If that happens, it's probably because of testing errors on our end that aren't your fault. Also, you don't have to worry about codeclimate saying there's 1 issue to fix. We will override that. Sorry if that was confusing!

If you need help with any of the above process, please leave a comment!

**Comment by akshay4n** [comment]
@noi5e sorry about the multiple pull requests.
The file has been edited. 
Thank you for your guidance!

**Comment by noi5e** [comment]
@akshay4n Thank you and no problem!! Thank you for hanging in there and making the changes so quickly. This PR is ready to merge! Someone else with merge access will have to do it, so let's just wait for someone to review it.

**Review by cesswairimu — APPROVED** [review]
Fantastic :tada:  thanks for working on this @akshay4n and many thanks to @noi5e for the guidance and reviews :tada: :tada:

**Pull request merged**
